runtime/doc/lua.txt

```diff
@@ -2970,6 +2979,27 @@ vim.fs.normalize({path}, {opts})                          *vim.fs.normalize()*
 
         vim.fs.normalize('$XDG_CONFIG_HOME/nvim/init.vim')
         -- '/Users/jdoe/.config/nvim/init.vim'
+
+        vim.fs.normalize('~/src/nvim/api/../tui/./tui.c')
+        -- '/home/jdoe/src/nvim/tui/tui.c'
+
+        vim.fs.normalize('.')
+        -- '.'
+
+        vim.fs.normalize('./foo/bar')
+        -- 'foo/bar'
+
+        vim.fs.normalize('foo/../../../bar')
+        -- '../../bar'
+
+        vim.fs.normalize('/home/jdoe/../../../bar')
+        -- Error: Invalid path, cannot resolve `..` past root directory
```

Comment:
I don't think this makes much sense together with "NOTE: In the root directory, resolving `..` will return the root directory itself.". `..` in root directory either errors or is resolved to the root itself.

Reply:
I was gonna comment on the same thing, I got confused by this as well.